java/code/src/com/redhat/rhn/frontend/xmlrpc/system/test/SystemHandlerTest.java

```diff
@@ -372,7 +372,6 @@ public void testSetChildChannelsDeprecated() throws Exception {
         SystemHandler mockedHandler = getMockedHandler();
         ActionChainManager.setTaskomaticApi(mockedHandler.getTaskomaticApi());
         SaltServerActionService sa = GlobalInstanceHolder.SALT_SERVER_ACTION_SERVICE;
-        sa.setCommitTransaction(false);
```

Comment:
I think this will change the test logic, since the transaction will be committed. I'm not sure about the side effects of this. Didn't you try to run all the test and see if it causes cross-test issues?